src/appMain/CMakeLists.txt

```diff
@@ -143,6 +143,12 @@ endif()
 add_executable(${PROJECT} ${SOURCES})
 target_link_libraries(${PROJECT} ${LIBRARIES})
 
+add_dependencies(${PROJECT} Policy)
+
+if(ENABLE_LOG)
+  add_dependencies(${PROJECT} install-3rd_party_logger)
```

Comment:
@orhan-mehmedov this explicit dependency ensures proper logger installation (configuration, appenders, etc.), correct me if I am wrong, but I don't see an issue to have it here.